validphys2/src/validphys/closuretest/multiclosure.py

```diff
@@ -21,23 +21,28 @@
     check_fits_underlying_law_match,
     check_fits_areclosures,
     check_fits_different_filterseed,
-    check_t0pdfset_matches_multiclosure_law
+    check_t0pdfset_matches_multiclosure_law,
 )
 from validphys.checks import check_use_t0
 
+import logging
+
+log = logging.getLogger(__name__)
+
 # bootstrap seed default
 DEFAULT_SEED = 9689372
 # stepsize in fits/replicas to use for finite size bootstraps
 SAMPLING_INTERVAL = 5
 
+
 # TODO: deprecate this at some point
 @check_fits_underlying_law_match
 @check_fits_areclosures
 @check_fits_different_filterseed
 @check_use_t0
 @check_t0pdfset_matches_multiclosure_law
 def internal_multiclosure_dataset_loader(
-    dataset, fits_pdf, multiclosure_underlyinglaw, fits, dataset_inputs_t0_covmat_from_systematics
+    dataset, fits_pdf, multiclosure_underlyinglaw, fits, t0_covmat_from_systematics
```

Comment:
Can you remind me the reason of using `t0_covmat_from_systematics` instead of the other?

Reply:
The `internal_multiclosure_dataset_loader` is meant for one dataset only and should not take the covariance matrix for whole DataGroup. 
There are more details in this closed PR:  https://github.com/NNPDF/nnpdf/pull/1719

Reply:
Ok thanks for the explanation!